docs/contrib/GitGuide.md

```diff
@@ -0,0 +1,207 @@
+# Git Quickstart
+
+This tutorial serves as a quickstart to Git and contributing to our repository. If you have not already set up OpenROAD, please follow the instructions [here](user/Build.md). 
+
+```{tip} This basic tutorial gives instruction for basic password Git authentication.
+If you would like to setup SSH authentication, please follow this [guide](https://help.github.com/set-up-git-redirect).
+```
+
+## Forking
+
+You will need your own fork to work on the code. Go to the `OpenROAD` project
+[page](https://github.com/The-OpenROAD-Project/OpenROAD) and hit the `Fork` button. You will
+want to clone your fork to your machine:
+
+```shell
+git clone https://github.com/your-user-name/OpenROAD.git
+cd OpenROAD
+git remote add upstream https://github.com/The-OpenROAD-Project/OpenROAD.git
+git fetch upstream
+```
+
+This creates the directory `OpenROAD` and connects your repository to
+the upstream (main project) *OpenROAD* repository.
+
+## Creating a branch
+
+You want your main branch to reflect only production-ready code, so create a
```

Comment:
```suggestion
You want your master branch to reflect only production-ready code, so create a
```

We still use "master" as the default branch. Let's change here and everywhere in this file just to avoid confusion.